bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/keys/show/ShowKeysUI.java

```diff
@@ -223,7 +223,7 @@ protected void configureShell(Shell newShell) {
 
 			Color color = JFaceResources.getColorRegistry().get(POPUP_COLOR_BG);
 			newShell.setBackground(color);
-			newShell.setAlpha(170);
+			newShell.setAlpha(100);
```

Comment:
Maybe now it's a little bit too transparent. I will test it locally later and check if I can come up with a better value.